CHANGELOG

```diff
@@ -0,0 +1,14 @@
+# Changelog
```

Comment:
This file should be called `CHANGELOG.md`. That'll ensure that things like GitHub reliably render the markdown formatting. Also, it's what the FAQ at keepachangelog.com recommends :-)